content/en/docs/tutorials/security/ns-level-pss.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Apply Pod Security Standards at the Namespace Level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters. 
+{{% /alert %}}
+
+Pod Security admission (PSA) is enabled by default in v1.23 and later, as it [graduated
+to beta](/blog/2021/12/15/pod-security-admission-beta/). Pod Security Admission
+is an admission controller that applies Pod Security Standards when pods are
+created. In this tutorial, you will enforce the `baseline` Pod Security Standard,
+one namespace at a time.
+
+# Pre-requisites
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+# Create cluster
+
+1. Create a `KinD` cluster as follows:
+
+    ```shell
+    kind create cluster --name psa-ns-level --image kindest/node:latest
+    ```
+    The output is similar to this:
+    ```
+    Creating cluster "psa-ns-level" ...
+     ✓ Ensuring node image (kindest/node:latest) 🖼 
+     ✓ Preparing nodes 📦  
+     ✓ Writing configuration 📜 
+     ✓ Starting control-plane 🕹️ 
+     ✓ Installing CNI 🔌 
+     ✓ Installing StorageClass 💾 
+    Set kubectl context to "kind-psa-ns-level"
+    You can now use your cluster with:
+    
+    kubectl cluster-info --context kind-psa-ns-level
+    
+    Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
+    ```
+
+2. Set the kubectl context to the new cluster
+    ```shell
+    kubectl cluster-info --context kind-psa-ns-level
+    ```
+    The output is similar to this:
+    ```
+    Kubernetes control plane is running at https://127.0.0.1:50996
+    CoreDNS is running at https://127.0.0.1:50996/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ```
+
+# Create Namespace
+
+Create a new namespace `example` for this tutorial:
+
+```shell
+kubectl create ns example
+```
+The output is similar to this:
+```
+namespace/example created
+```
+
+# Applying one Pod Security Standard
```

Comment:
I don't see the value in instructions for both one and multiple Standards. Make a single section called "Apply Pod Security Standards" and keep the instructions for applying multiple.

Reply:
I'd be OK with adjusting this post-merge

Reply:
Merged the two sections together!